Support/Scripts/run-lldb-tests.sh

```diff
@@ -130,7 +130,7 @@ else
   export LLDB_DEBUGSERVER_PATH="$build_dir/ds2"
 fi
 
-export LLDB_TEST_TIMEOUT=45m
+export LLDB_TEST_TIMEOUT=8m
```

Comment:
From the issue: "TestLldbGdbServer was flaky with the default 10 minute timeout"

That's kind of funny, since this test failed recently :) Perhaps the timeout was due to the random failure, and not to a random runtime.